Microsoft.Toolkit.Uwp.UI.Media/Geometry/Core/PathElementFactory.cs

```diff
@@ -0,0 +1,203 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text.RegularExpressions;
+using Microsoft.Toolkit.Uwp.UI.Media.Geometry.Elements.Path;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Geometry.Core
+{
+    /// <summary>
+    /// Factory class to instantiate various PathElements.
+    /// </summary>
+    internal static class PathElementFactory
+    {
+        /// <summary>
+        /// Creates a default Path Element for the given PathFigureType
+        /// </summary>
+        /// <param name="figureType">PathFigureType</param>
+        /// <returns>ICanvasPathElement</returns>
+        internal static ICanvasPathElement CreateDefaultPathElement(PathFigureType figureType)
+        {
+            ICanvasPathElement result;
+
+            switch (figureType)
+            {
+                case PathFigureType.FillRule:
+                    result = new FillRuleElement();
+                    break;
+                default:
+                    throw new ArgumentException("Creation of Only Default FillRuleElement is supported.", nameof(figureType));
+            }
+
+            return result;
```

Comment:
Converting it to the modern switch statement and adding a ThrowHelper.Throw___ does not compile as the return type of ThrowHelper APIs is void.

![image](https://user-images.githubusercontent.com/7021835/104862520-40568100-58e8-11eb-9cb5-fb8864b1e54b.png)

Since this check has to be for only one PathFigureType, I converted it to a if-else condition instead.